### PR TITLE
Fix typo in README.md

### PR DESCRIPTION
**Pull request by qhua948**
Pretty sure there is a mistake in the README file

### CODE REVIEW

File: tags [hunk 1]
```diff
@@ -0,0 +1,108 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
+!_TAG_FILE_SORTED	1	/0=unsorted, 1=sorted, 2=foldcase/
+!_TAG_PROGRAM_AUTHOR	Darren Hiebert	/dhiebert@users.sourceforge.net/
+!_TAG_PROGRAM_NAME	Exuberant Ctags	//
+!_TAG_PROGRAM_URL	http://ctags.sourceforge.net	/official site/
+!_TAG_PROGRAM_VERSION	5.8	//
+APP_BASE_NAME	gradlew.bat	/^set APP_BASE_NAME=%~n0$/;"	v
+APP_HOME	gradlew.bat	/^set APP_HOME=%DIRNAME%$/;"	v
```
Comment (markGilchrist) [inline tags]: @qhua948  did you mean to include this in you pr?

File: README.md [hunk 2]
```diff
@@ -296,7 +296,7 @@ Fuel.get("http://httpbin.org/get", listOf("foo" to "foo", "bar" to "bar")).respo
 }
 
 Fuel.delete("http://httpbin.org/delete", listOf("foo" to "foo", "bar" to "bar")).response { request, response, result ->
-    //resolve to http://httpbin.org/get?foo=foo&bar=bar
+    //resolve to http://httpbin.org/delete?foo=foo&bar=bar
```
Comment (markGilchrist) [inline README.md]: I think you are right

File: tags (hunk 1 above)
Comment (qhua948) [reply tags]: Sorry, it was included by ctag, I'll remove it.

Comment (markGilchrist) [reply tags]: perfect then I will approve

**Comment by kittinunf** [comment]
Merge! 👍

**Pull request merged**